cake/cake_fitting.py

```diff
@@ -157,7 +101,7 @@ def half_life_calc(y_data0, x_data, y_data):
     return half_life
 
 
-def est_k_order(x, y, t0, r0, cat_add_rate, half_life):
+def est_k_order(r_ord, cat_ord, t_del, r0, cat_add_rate, t_inj, half_life):
```

Comment:
New parameter t_inj.  How does this differ from t0? Also, are the renamings simply semantic or do the variables take on different meanings now?

Reply:
it's the time of the beginning of catalyst injection, and now t_del is the time after catalyst injection where the reaction begins. The catalyst injection time is know, t_del may not be known but can be fit.